tealium_angular.js

```diff
@@ -57,7 +57,10 @@ angular.module("TealiumHelper", ["TealiumHelper.data"])
     addViewIdMapEntry: tealiumDataProvider.addViewIdMapEntry,
     $get: [ "tealiumData", "$location", function(tealiumData, $location) {
       if (!config.account || !config.profile) {
-        throw new Error("account or profile value not set.  Please configure Tealium first");
+        if (config.account || config.profile) console.warn("Tealium configuration not valid. Please check your account and profile");
```

Comment:
Not sure if this code has been tested.  I'd expect to see the if statement to match the previous if statement logic.

Reply:
This if statement is just for the warning, line 59 matches the previous statement logic. The thinking there was just to only warn if the config object is *incomplete* but just not initialise Tealium if the config object *didn't exist at all*. It means that you need to configure Tealium for it to be loaded in, which seems like nice behaviour to me.

Happy to remove that if statement around the warning in this PR if that's the preference? We'll likely just need to keep it in our fork so it doesn't clutter up websites who don't use the integration.

Code has been tested!

Reply:
I would expect the if statement to be the same (as follows)

```if (!config.account || !config.profile)```

instead I see this

```if (config.account || config.profile)```

Reply:
The logic is line 59 (https://github.com/Tealium/integration-angularjs/pull/30/files#diff-904cf4402c617889ebae2f52f6e77c4dR59) which is unchanged. That's why it's not showing in the diff.

Line 59 uses `!config.account || !config.profile`. Within that conditional we then have L60 which has the positive condition for the warning. The `if (config.account || config.profile)` only effects whether the warning is shown. Regardless, there's a return after it which will halt the function (from Line 61). That return will fire based on the original conditional